src/test/java/com/trainer/name/integrationtest/TrainerRestApiIntegrationTest.java

```diff
@@ -89,4 +90,24 @@ public class TrainerRestApiIntegrationTest {
                 "{\"id\":1,\"name\":\"ゼイユ\",\"email\":\"Zeiyu498@merry.bluebe\"}",
                 response, JSONCompareMode.STRICT);
     }
+
+    @Test
+    @DataSet(cleanBefore = true, cleanAfter = true) // テスト前後にデータをクリーンアップ
+    @Transactional
+    void 新しいトレーナーが作成されること() throws Exception {
+        // 作成する新しいトレーナーの情報を指定
+        String newTrainerRequest = "{\"name\":\"新しいトレーナー\",\"email\":\"newtrainer@example.com\"}";
```

Comment:
Javaのテキストブロックを使えばよいかと！
```suggestion
        String newTrainerRequest = """
          {
            "key": "value"
          }
        """;
```
このように JSON をフォーマットして定義できますし、 \ によるエスケープも不要です。

Reply:
44cb7ba6059c88fe089d54af177aeb151b31ff78
Javaのテキストブロックを使用しました
![image](https://github.com/Ito-0131/10th-Assignment/assets/138368847/92322889-9f41-4eed-b3e9-5bb58e4ac00c)